src-java/rule-manager/rule-manager-api/src/main/java/org/openkilda/action/SetFieldAction.java

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.action;
+
+import org.openkilda.Field;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@JsonSerialize
+@Builder
+public class SetFieldAction implements Action {
+
+    long value;
```

Comment:
are we going to send ipv4 and eth addresses as just long?

Reply:
Yes, main idea was to send to speaker raw data if possible.